integrated_channels/degreed2/client.py

```diff
@@ -275,6 +277,36 @@ def create_content_metadata(self, serialized_data):
                 f'Degreed2APIClient create_content_metadata failed with status {status_code}: {response_body}',
                 status_code=status_code
             )
+        # once course is created successfully, we need to do 2 more steps
+        # 1. Fetch skills from enterprise-catalog
+        client = EnterpriseCatalogApiClient()
+        metadata = client.get_content_metadata_content_identifier(
+            enterprise_uuid=self.enterprise_configuration.enterprise_customer.uuid,
+            content_id=external_id
+        )
+        LOGGER.info(
+            generate_formatted_log(
+                self.enterprise_configuration.channel_code(),
+                self.enterprise_configuration.enterprise_customer.uuid,
+                None,
+                None,
+                f'[Degreed2Client] metadata: {metadata}'
+            )
+        )
+        # 2. Transmit to degreed
+        skills = metadata['skill_names']
+        if skills:
+            self.assign_course_skills(external_id, skills)
+            LOGGER.info(
+                generate_formatted_log(
+                    self.enterprise_configuration.channel_code(),
+                    self.enterprise_configuration.enterprise_customer.uuid,
+                    None,
+                    None,
+                    f'[Degreed2Client] transmitted skills: {metadata["skill_names"]},'
+                    f'for course: {external_id}'
+                )
+            )
```

Comment:
We already are logging something similar in the assign_course_skill method so this log can be removed as well.